package.json

```
@@ -46,7 +46,7 @@
     ]
   },
   "engines": {
-    "node": ">=16 <=18"
+    "node": ">=18 <=22"
```

Update the Boemly package first!
```suggestion
    "node": ">=18 <=22"
```